### PR TITLE
Feature/oauth2.0

### PR DESCRIPTION
**Pull request by AlekseyKostenko**

### CODE REVIEW

File: packages/asc-web-common/api/people/index.js [hunk 1]
```diff
@@ -197,6 +197,29 @@ export function updateUserType(type, userIds) {
   });
 }
 
+export function thirdPartyLinkAccount(SerializedProfile) {
```
Comment (AlexeySafronov) [inline packages/asc-web-common/api/people/index.js:200]: Rename to linkThirdPartyAccount

File: packages/asc-web-common/api/people/index.js [hunk 2]
```diff
@@ -197,6 +197,29 @@ export function updateUserType(type, userIds) {
...
+  return request({
+    method: "put",
+    url: "people/thirdparty/linkaccount.json",
+    data: { SerializedProfile },
+  });
+}
+
+export function thirdPartySignup(signupAccount) {
```
Comment (AlexeySafronov) [inline packages/asc-web-common/api/people/index.js:208]: Rename  to signupThirdParty

File: packages/asc-web-common/api/people/index.js (hunk 1 above)
Comment (AlexeySafronov) [reply packages/asc-web-common/api/people/index.js:200]: Rename SerializedProfile to profile

File: packages/asc-web-common/api/people/index.js (hunk 2 above)
Comment (AlexeySafronov) [reply packages/asc-web-common/api/people/index.js:208]: Rename signupAccount to account

File: packages/asc-web-common/api/people/index.js [hunk 3]
```diff
@@ -197,6 +197,29 @@ export function updateUserType(type, userIds) {
...
+  return request({
+    method: "post",
+    url: "people/thirdparty/signup.json",
+    data: signupAccount,
+  });
+}
+
+export function thirdPartyUnlinkAccount(provider) {
```
Comment (AlexeySafronov) [inline packages/asc-web-common/api/people/index.js:216]: Rename thirdPartyUnlinkAccount to unlinkThirdPartyAccount

File: packages/asc-web-common/api/user/index.js [hunk 4]
```diff
@@ -13,6 +13,14 @@ export function login(userName, passwordHash) {
   });
 }
 
+export function thirdPartyLogin(SerializedProfile) {
```
Comment (AlexeySafronov) [inline packages/asc-web-common/api/user/index.js:16]: Rename thirdPartyLogin to LoginThirdParty 

Rename SerializedProfile to profile

File: packages/asc-web-common/store/AuthStore.js [hunk 5]
```diff
@@ -154,6 +156,22 @@ class AuthStore {
     }
   };
 
+  thirdPartyLogin = async (SerializedProfile) => {
```
Comment (AlexeySafronov) [inline packages/asc-web-common/store/AuthStore.js:159]: Rename thirdPartyLogin to loginThirdParty

Rename SerializedProfile to profile

File: public/thirdparty/third-party.html [hunk 6]
```diff
@@ -82,17 +82,8 @@
...
     </script>
   </head>
-  <body>
-    <div class="offline-container">
-      <div class="offline-disclaimer">
-        <span class="offline-title">Third party Page</span>
-      </div>
-    </div>
-  </body>
+  <body></body>
```
Comment (AlexeySafronov) [inline public/thirdparty/third-party.html:88]: Add handling errors

File: packages/asc-web-common/constants/index.js [hunk 7]
```diff
@@ -121,9 +126,7 @@ export const providersData = Object.freeze({
     label: "SignInWithLinkedIn",
     icon: "/static/images/share.linkedin.react.svg",
   },
-});
-
-export const i18nBaseSettings = {
+});export const i18nBaseSettings = {
```
Comment (AlexeySafronov) [inline packages/asc-web-common/constants/index.js:129]: Restore spaces

File: packages/asc-web-common/api/people/index.js (hunk 1 above)
Comment (AlekseyKostenko) [reply packages/asc-web-common/api/people/index.js]: Fixed

**Review by AlexeySafronov — APPROVED** [review]

**Pull request merged**